app/i18n/locales/de-DE.json

```diff
@@ -0,0 +1,273 @@
+  "api.errors.updateAdaTxsHistoryError": "Fehler beim Aktualisieren des ADA Transaktions-Verlaufes.",
+  "api.errors.updateAdaWalletError": "Fehler beim Aktualisieren der ADA Wallet.",
+  "app.errors.unknowError": "unbekannter Fehler.",
+  "daedalusTransfer.error.noTransferTxError": "Es gibt keine Transaktion zum versenden.",
+  "daedalusTransfer.error.transferFundsError": "Es ist nicht möglich den Geldwert zu übertragen.",
+  "daedalusTransfer.error.webSocketRestoreError": "Fehler beim Wiederherstellen der Blockchain-Adressen.",
+  "daedalusTransfer.errorPage.backButton.label": "Zurück",
+  "daedalusTransfer.errorPage.title.label": "Es ist nicht möglich die Daedalus Wallet wieder herzustellen.",
+  "daedalusTransfer.form.instructions.step0.text": "Bitte die 12 Wiederherstellungs-Kennwörter der Daedalus-wallet eingeben, um den Kontostand von Daedalus zu Yoroi zu übertragen.",
+  "daedalusTransfer.instructions.attention.confirmation": "Den gesamten Kontostand vom Daedalus Wallet übernehmen",
+  "daedalusTransfer.instructions.attention.text": "Yoroi und Daedalus wallets verwenden unterschiedliche Schlüssel-Ableitungs-Schemas und haben ein eigenes Adressformat. Aus diesem Grund kann nicht mit der Daedalus-Wallet weiter gemacht werden. Statt dessen muss eine Yoroi-Wallet erstellt und der Kontostand übertragen werden (In Zukunft wird das nicht mehr so sein). Daedalus und Yoroi wallets sind voll kompatibel um Geldwerte zu übertragen. Wenn Sie keine Daedalus-Instanz mit ihrer Wallet zur Verfügung haben, können sie die 12 Wiederherstellungs-Wörter benutzen um den Kontostand von Daedalus zu Yoroi zu übertragen.",
+  "daedalusTransfer.summary.addressFrom.subLabel": "Daedalus wallet Adressen",
+  "daedalusTransfer.title": "Übernehme den Kontostand von Daedalus",
+  "daedalusTransfer.waiting.checkingAddresses.generatingTx": "Erstelle die Übertragungs-Transaktion",
+  "daedalusTransfer.waiting.progressInfo.checkingAddresses": "Überprüfe den Kontostand der Adressen",
+  "daedalusTransfer.waiting.progressInfo.restoringAddresses": "Adressen abrufen",
+  "daedalusTransfer.waiting.title.label": "Daedalus wallet wird wieder hergestellt",
+  "environment.apiName.cardano": "Cardano",
+  "environment.apiVersion.cardano": "1.0.4",
+  "environment.currency.ada": "ADA",
+  "global.errors.fieldIsRequired": "Dieses Feld muss ausgefüllt werden.",
+  "global.errors.invalidRepeatPassword": "Stimmt nicht überein.",
+  "global.errors.invalidWalletName": "Der Wallet Namen muss aus mindestens 1 und maximal 40 Zeichen bestehen.",
+  "global.errors.invalidWalletPassword": "Ungültiges Kennwort",
+  "global.labels.cancel": "Abbrechen",
+  "global.labels.change": "Wechseln",
+  "global.labels.create": "Erstelle",
+  "global.labels.remove": "Entferne",
+  "global.labels.save": "Speichern",
+  "global.language.chinese.simplified": "Chinesisch vereinfacht (简体中文)",
```

Comment:
we would like to match the current translations, so every language would be written in its own language